rdflib/plugins/sparql/algebra.py

```diff
@@ -761,7 +761,7 @@ def translateUpdate(q, base=None, initNs=None):
 
         res.append(translateUpdate1(u, prologue))
 
-    return res
+    return Update(prologue, res)
```

Comment:
Just a technical note, similar to what I raised in #1626 

This technically changes the RDFLib API, I'm not sure that we consider the methods in here part of the public API, but I think if we don't we should make a note in the docs. Happy to do that if there is agreement on it.

Reply:
> This technically changes the RDFLib API, I'm not sure that we consider the methods in here part of the public API, but I think if we don't we should make a note in the docs. Happy to do that if there is agreement on it.

I guess from a pragmatic perspective, "public API" is anything that *can* be imported and the SPARQL API is very sparsely documented but yes, agreed.

Reply:
I added it to me my to-do list to note these methods are not considered part of the public interface and may break compatibility within a single major version and will do it in a separate PR.

Reply:
Perhaps it's breaking the public API but it's making it more consistent so I suppose we just release a slightly more significant version to include this, perhaps 6.2.0.